chainio/utils/bindings.go

```diff
@@ -90,6 +90,7 @@ type AvsRegistryContractBindings struct {
 	StakeRegistryAddr          gethcommon.Address
 	BlsApkRegistryAddr         gethcommon.Address
 	OperatorStateRetrieverAddr gethcommon.Address
+	IndexRegistryAddr          gethcommon.Address
```

Comment:
You only added the addr. You need to also add the bindings for the IndexRegistry.

Reply:
Oh yes, just added, sorry